workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts

```diff
@@ -390,3 +390,23 @@ type UserAccount = {
 	isPublisherAccount: boolean;
 	password: string;
 };
+
+type RequestBody = {
```

Comment:
You can try something like
```suggestion
type RequestBody = {[keys: string]: string}
```